coding-concepts.js

```diff
@@ -5,47 +5,47 @@
 // First, predict the outcome based on your understanding of the code.
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
-// --------------------1) What will this log?
-
-var cohort = "Delta 2021"
-// console.log(cohort.length)
-
-// a) Your answer:
-// b) Verify and explain:
+// // --------------------1) What will this log?
+//
+// var cohort = "Delta 2021"
+// // console.log(cohort.length)
+//
+// // a) Your answer: 10
+// b) Verify and explain: There are 10 digits in the string "Delta 2021" since the space is included.
 
 
 // --------------------2) What will this log?
 
-var greeting = "Hello World!"
+// var greeting = "Hello World!"
 // console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: L
+// b) Verify and explain: L is what it logs due to the fact that L is in position 3 within the string "Hello World"
 
 
 // --------------------3) What will this log?
-
-var languages = ["JavaScript", "Ruby", "Python", "C++"]
-var index = 1
+//
+// var languages = ["JavaScript", "Ruby", "Python", "C++"]
+// var index = 1
 // console.log(languages[index])
-
-// a) Your answer:
-// b) Verify and explain:
-
-
-// --------------------4) What will this log?
-
-var weekendDays = ["saturday", "sunday"]
+//
+// // a) Your answer: Ruby
+// // b) Verify and explain: the string "Ruby" is in the 1 position in the string and that is what the variable "index" is calling for in the console.log();
+//
+//
+// // --------------------4) What will this log?
+//
+// var weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
-
-
-// --------------------5) What will this log?
-
-var dataTypes = ["number", "string", "Boolean"]
+// // a) Your answer: Changes string to uppercase
+// // b) Verify and explain: TypeError: weekendDays.toUpperCase is not a function. I am not sure why this logs. I am assuming that it is related to the variable being attached to the string incorrectly.
```

Comment:
close! its because .toUpperCase() only works on strings while weekendDays is an array that holds strings!

It's like eating the plastic baggie around your sandwich!